packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1ERC20Gateway.sol

```diff
@@ -75,6 +76,30 @@ contract L1ERC20Gateway is L1ArbitrumExtendedGateway {
             );
     }
 
+    // note: reentrancy here
```

Comment:
would be good to expand on this comment explaining the concern
also use natspec format https://docs.soliditylang.org/en/develop/natspec-format.html

Reply:
Doesn't seems to have reentrancy risk here tho given the state changing logic are in 
`outboundTransferCustomRefund` with nonReentrant.